src/lotus_json/registered_po_st_proof.rs

```diff
@@ -13,15 +13,15 @@ impl HasLotusJson for RegisteredPoStProof {
     fn snapshots() -> Vec<(serde_json::Value, Self)> {
         vec![(
             json!(0),
-            RegisteredPoStProof::from(RegisteredPoStProofV3::StackedDRGWinning2KiBV1),
+            RegisteredPoStProof::from(RegisteredPoStProofV4::StackedDRGWinning2KiBV1),
```

Comment:
Not sure if it's a good idea to use fvm_shared_latest::sector::RegisteredPoStProof here so that we won't miss it with the next fvm upgrade

Reply:
It's not a real issue, given enum discriminants for the proofs remain the same. That said, it'd make sense to get rid of fvmX occurrences from outside of `shim` and `fvmX` modules. I'll address it in a follow-up.